tlatools/org.lamport.tlatools/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -45,7 +45,7 @@ org.eclipse.jdt.core.compiler.problem.missingEnumCaseDespiteDefault=disabled
 org.eclipse.jdt.core.compiler.problem.missingHashCodeMethod=ignore
 org.eclipse.jdt.core.compiler.problem.missingOverrideAnnotation=ignore
 org.eclipse.jdt.core.compiler.problem.missingOverrideAnnotationForInterfaceMethodImplementation=enabled
-org.eclipse.jdt.core.compiler.problem.missingSerialVersion=warning
+org.eclipse.jdt.core.compiler.problem.missingSerialVersion=ignore
```

Comment:
I endorse this change. 👍 

I am no expert on Java's serialization framework, but my current understanding is that
 - [the automatically-generated `serialVersionUID` suffices](https://github.com/projectlombok/lombok/wiki/WHY-NOT:-serialVersionUID)
 - long-term we should be moving away from Java's serialization framework (historically it has been riddled with security holes and other sharp edges---even at Oracle our internal guidance is to steer well clear of this feature)

(But @lemmy you might want to weigh in if you have any additional thoughts.)

Reply:
I would have preferred for this to be a dedicated commit, but I have no objections to ignoring this warning.